saga-web/src/main/resources/saga-frontend/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "saga-frontend",
+  "version": "2.3.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/akveo/ngx-admin.git"
+  },
+  "bugs": {
+    "url": "https://github.com/akveo/ngx-admin/issues"
+  },
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "IE 11"
+  ],
+  "scripts": {
+    "ng": "ng",
+    "conventional-changelog": "conventional-changelog",
+    "start": "ng serve --port 5200",
+    "build": "ng build",
+    "build:prod": "npm run build -- --prod --aot",
+    "test": "ng test",
+    "test:coverage": "rimraf coverage && npm run test -- --code-coverage",
+    "lint": "ng lint",
+    "lint:fix": "ng lint saga-frontend --fix",
+    "lint:styles": "stylelint ./src/**/*.scss",
+    "lint:ci": "npm run lint && npm run lint:styles",
+    "pree2e": "webdriver-manager update --standalone false --gecko false",
+    "e2e": "ng e2e",
+    "docs": "compodoc -p src/tsconfig.app.json -d docs",
+    "docs:serve": "compodoc -p src/tsconfig.app.json -d docs -s",
+    "prepush": "npm run lint:ci",
+    "release:changelog": "npm run conventional-changelog -- -p angular -i CHANGELOG.md -s"
+  },
+  "dependencies": {
+    "@agm/core": "1.0.0-beta.2",
+    "@angular/animations": "6.0.0",
+    "@angular/common": "6.0.0",
+    "@angular/compiler": "6.0.0",
+    "@angular/core": "6.0.0",
+    "@angular/forms": "6.0.0",
+    "@angular/http": "6.0.0",
+    "@angular/platform-browser": "6.0.0",
+    "@angular/platform-browser-dynamic": "6.0.0",
+    "@angular/router": "6.0.0",
+    "@asymmetrik/ngx-leaflet": "3.0.1",
+    "@nebular/auth": "2.0.0-rc.10",
+    "@nebular/security": "2.0.0-rc.10",
+    "@nebular/theme": "2.0.0-rc.10",
+    "@ng-bootstrap/ng-bootstrap": "1.0.0",
+    "@swimlane/ngx-charts": "7.0.1",
+    "angular-tree-component": "7.2.0",
+    "angular2-chartjs": "0.4.1",
+    "angular2-toaster": "6.0.0",
+    "bootstrap": "4.0.0",
+    "chart.js": "2.7.1",
+    "ckeditor": "4.7.3",
+    "classlist.js": "1.1.20150312",
+    "core-js": "2.5.1",
+    "echarts": "^4.0.2",
+    "intl": "1.2.5",
+    "ionicons": "2.0.1",
+    "leaflet": "1.2.0",
+    "nebular-icons": "1.0.9",
+    "ng2-ckeditor": "1.1.13",
+    "ng2-completer": "2.0.8",
+    "ng2-smart-table": "1.3.5",
+    "ngx-echarts": "2.0.1",
+    "normalize.css": "6.0.0",
+    "pace-js": "1.0.2",
+    "roboto-fontface": "0.8.0",
```

Comment:
are we using this anywhere in our code?

Reply:
the charts library is required as we will be implementing some charts in the UI as per the original requirements.
Normalize.css makes browsers render all elements more consistently and in line with modern standards.
pace.js is used for the nice page load bars that appear on the top of the page.
roboto fontface is used across the UI.